components/script/task_manager.rs

```diff
@@ -0,0 +1,130 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Since this is a completely new file, you can safely run `./mach rustfmt -d  components/script/task_manager.rs` and apply the changes to the commit (requires installing first if you haven't, see https://github.com/rust-lang-nursery/rustfmt)